pkg/webhook/cluster.go

```diff
@@ -21,6 +21,8 @@ const (
 	openshiftCAPINamespace = "openshift-cluster-api"
 )
 
+var errClusterDeletionNotAllowedInOpenShiftCAPINamespace = errors.New("deletion of cluster is not allowed in openshift-cluster-api namespace")
```

Comment:
I think you could compose this error string with a fmt.Sprintf internally so you can reference `openshiftCAPINamespace` instead of hardcoding it